roster.md

```diff
@@ -1,12 +1,18 @@
 # Intern Roster
 
+<<<<<<< HEAD
 ### Amonte Andrews (no team)
 ``` js
 var amonte = {
-    niceGuy:true,
-    covidHobbies:['Dark Season 3', 'Coding', 'Reading'],
+=======
+## tanya (no team)
+``` js 
+var tanya = {
+>>>>>>> 2e90c4cf45b9e41fba6462c10e42437c50ab1d82
+    BackEndDevTeam:true,
+    covidHobbies:['Piano', 'Spending Time With Family', 'Reading'],
     getExperience: function(andStrengths) {
-        return !andStrengths ? "Posted in slack" : "Still posted in slack.";
+    strengths:[Hardworking, Friendly, Brave];
     }
 }
```

Comment:
When you merged upstream/development into your branch, you needed to manually fix the conflicts that created in roster.md. You see the >>>>>> and <<<<<<  and =======. Those are sections that git was tell you need some modifications. DM with questions.